spec/factories/cars.rb

```diff
@@ -1,11 +1,11 @@
 FactoryBot.define do
   factory :car do
-    user_id { create(:correct_user_update).id }
-    manufacturer { 'test_manufacturer' }
-    model { 'test_model' }
-    year { 2000 }
-    color { '#324' }
-    license_plat_number { '2342' }
+    association :user, factory: [:user, :correct_user_update]
```

Comment:
why you can't just do like here:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations
?

Reply:
Because in the previous version of the user's factory, by default it was created not valid